doc/holochain_101/src/first_steps.md

```diff
@@ -205,7 +205,7 @@ fn handle_create_list(list: List) -> ZomeApiResult<Address> {
 
 The `hdk::commit_entry` function is how a zome can interact with holochain core to add entries to the DHT or local chain. This will trigger the validation function for the entry and if successful will store the entry and return its hash/address.
 
-The `add_item` function requires the use of holochain links to associate two entries. In holochain-proto this required the use of a commit with a special Links entry but it can now be done using the HDK function `link_entries(address1, address2, tag)`. The add item handler accepts a `ListItem` and an address of a list, commits the `ListItem`, then links it to the list address:
+The `add_item` function requires the use of holochain links to associate two entries. In holochain-proto this required the use of a commit with a special Links entry but it can now be done using the HDK function `link_entries(address1, address2, link_type)`. The add item handler accepts a `ListItem` and an address of a list, commits the `ListItem`, then links it to the list address:
```

Comment:
Missing the `link_tag` parameter in `link_entries`?  Also I think it would be good to increase the documentation here to explain the difference between type and tag.  I.e. that instead of creating a new entry type (which we needed for figuring out which validation to run) that's now encoded in the `link_type` string passed directly to the `link_entries` funciton.